### ISSUE TITLE
Dark vs Light theme selection

### ISSUE THREAD

**Issue by ganomi46** [halogenica/beautifulhugo#512]
Would it be possible to have light theme as default and button to allow the visitor to change the theme. I received a lot of feedback on why the theme is dark. It seem the automatic change by the browser is not very welcome. Thx

**Comment by ripienaar** [comment]
Strong agree, I am a huge fan of dark themes and set everything possible to dark - but this one is just not good and it's ruined beautifulhugo for me.

**Comment by Xhanti** [comment]
I also agree. It threw me a bit not being able to change it to light mode. It would be great to allow the user to control it.

**Comment by henryiii** [comment]
IMO, this is really a problem with browsers. They should provide a toggle to override the system setting, ideally per webpage. Asking webpages to all add custom toggles seems wasteful and silly. But I'd be fine with a well-implemented selector. Improvements to the colors in dark mode would be welcome as well if there's an issue with it being "not good".

**Comment by henryiii** [comment]
(#500 doubles the number of CSS files and doesn't add a toggle, it just lets the site author select a "non-changing light-only" theme, basically.)

**Comment by hydr0nium** [comment]
Yes I agree with @henryiii that it's a browser problem, but I have also been a big advocate for a change theme button. I am not really experienced with CSS, HTML much so I would be happy if someone could implement that feature. I would also be down to help as much as I can
